test/extensions.test.tsx

```diff
@@ -36,8 +35,7 @@ describe("extensions", () => {
           First, solve the problem. Then, write the code.
         </MjmlConditionalComment>
       );
-      const markup = renderToMjml(comment);
-      expect(markup).toBe(
+      expect(renderToMjml(comment)).toBe(
```

Comment:
I know it's a little outside the scope of this library, but I'm wondering if we should `renderToHtml`.
The downside is that tests will sometimes break on an MJML upgrade.
The upside is that we see what a test's output really would be.
We would have to figure out the skeleton.

Probably not worth it at the moment, but figured I would make the suggestion

Reply:
I thought about this as well. Definitely something we should discuss further